queries.sql

```diff
@@ -5,4 +5,22 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu'; --
 SELECT name, escape_attempts FROM animals WHERE weight_in_kg > 10.5; -- List name and escape attempts of animals that weigh more than 10.5kg
 SELECT * FROM animals WHERE neutered = TRUE; -- Find all animals that are neutered.
 SELECT * FROM animals WHERE NOT name = 'Gabumon'; -- Find all animals not named Gabumon.
-SELECT * FROM animals WHERE weight_in_kg >= 10.4 AND weight_in_kg <= 17.3; -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
\ No newline at end of file
+SELECT * FROM animals WHERE weight_in_kg >= 10.4 AND weight_in_kg <= 17.3; -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+
+SELECT COUNT(*) AS totalAnimals FROM animals; -- How many animals are there?
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0; -- How many animals have never tried to escape?
```

Comment:
- [ ] To get the number of animals who have never tried to escape, the `WHERE` statement should be `escape_attempts = 0`. Kindly modify this.